executor/statement_context_test.go

```diff
@@ -37,9 +36,7 @@ func (s *testSuite) TestStatementContext(c *C) {
 
 	tk.MustExec(strictModeSQL)
 	tk.MustQuery("select * from sc where a > cast(1.1 as decimal)").Check(testkit.Rows("2"))
-	_, err := tk.Exec(`select * from sc where a > cast(1.1 as decimal);
-		        update sc set a = 4 where a > cast(1.1 as decimal)`)
-	c.Check(terror.ErrorEqual(err, types.ErrTruncated), IsTrue)
+	tk.MustExec("update sc set a = 4 where a > cast(1.1 as decimal)")
```

Comment:
I've tested that this behavior confirms to MySQL 5.6, 5.7...
The previous code is wrong.